### DISCUSSION TITLE
Filter what can be selected in relational fields (published items or another field:value constraint)

### DISCUSSION THREAD

**Discussion by jjvanrossum** [directus/directus#2804] [Archive]
It would be nice to have the ability to make sure only certain items can be selected in a relational field.

For instance

- only the published ones or

- only the ones where field=value

I have the case of several forms where certain forms should be selectable for collection a and other forms collectable for b

**Comment by benhaynes** [comment]
Ohh, I like this. I remember wanting this a while ago, but forgot about it. Essentially it would mean adding a "filter" option to the relational interface. This option would allow you to limit selectable items to a smaller, filtered subset.

**Reply by jjvanrossum** [reply]
Exactly. 

Although it does make your relation more complex so you might want to have an info button or something that it is a filtered collection.
